frontend/src/concepts/analyticsTracking/initSegment.tsx

```diff
@@ -0,0 +1,80 @@
+import { DEV_MODE } from '~/utilities/const';
+
+export const initSegment = async (props: {
+  segmentKey: string;
+  enabled: boolean;
+}): Promise<void> => {
+  const { segmentKey, enabled } = props;
+  const analytics = (window.analytics = window.analytics || []);
+  if (analytics.initialize) {
+    if (DEV_MODE) {
+      /* eslint-disable-next-line no-console */
+      console.trace('============= Windows.analytics not set ==========');
```

Comment:
😕 I'm not sure I understand this scenario... can you help me out here? If `Windows.analytics` is not set, how are you getting `window.analytics.initialize` to be truthy?